src/modules/database/database.js

```diff
@@ -1,3 +1,16 @@
-export let users = [];
+import pg from "pg";
+
+const { Pool } = pg;
+
+const pool = new Pool({
+  user: "postgres",
+  host: "localhost",
+  database: "users",
+  password: "12345678",
```

Comment:
Secrets should be in your `.env`. This is a MAJOR security issue.

Reply:
Fixed